hastexo_guacamole_client/consumers.py

```diff
@@ -20,8 +20,8 @@ async def connect(self):
         """
         Initiate the GuacamoleClient and create a connection to it.
         """
-        guacd_hostname = os.getenv('GUACD_HOSTNAME', 'localhost')
-        guacd_port = os.getenv('GUACD_PORT', 4822)
+        guacd_hostname = os.getenv('GUACD_SERVICE_HOST', 'guacd')
+        guacd_port = os.getenv('GUACD_SERVICE_PORT', 4822)
```

Comment:
This is unrelated to the change of variable names, but are we sure that setting the port would ever have worked? Environment variables are strings.

I think you may want to fix this to:

```python
guacd_port = int(os.getenv('GUACD_SERVICE_PORT', '4822'))
```

Reply:
At the moment I cannot recall when did we use the defaults and not. In any case, it makes sense to correct it, I'll update it now, thanks! :)